qt/python/mantidqt/widgets/plotconfigdialog/imagestabwidget/view.py

```diff
@@ -100,16 +97,30 @@ def set_reverse_colormap(self, checked):
         self.reverse_colormap_check_box.setCheckState(qt_checked)
 
     def get_min_value(self):
-        return self.min_value_spin_box.value()
+        min_value = self.min_value_line_edit.text()
+        try:
+            return float(min_value)
+        except ValueError:
+            logger.warning(f"Failed to set min value to '{min_value}'. Using '{self.min_value}' instead.")
```

Comment:
Could I ask why we don't trap the errors on setting the value so that it can't get to this state?

Reply:
Good point, I shouldn't have been copying the axes tab